Sources/Components/Label/Label.swift

```diff
@@ -0,0 +1,146 @@
+import SwiftUI
+
+extension Warp {
+    /// A customizable label component used to display a title, optional additional information, and an info tooltip.
+    ///
+    /// The `Warp.Label` component displays a main title with optional additional information. It can also show an info icon which, when tapped, presents custom tooltip content provided by the user.
+    ///
+    /// **Usage:**
+    ///
+    /// ```swift
+    /// Warp.Label(
+    ///     title: "Label Title",
+    ///     additionalInformation: "Optional Info",
+    ///     showToolTipImage: true,
+    ///     tooltipContent: AnyView(Warp.Tooltip(title: "Tooltip text"))
+    /// )
+    /// ```
+    ///
+    /// - Parameters:
+    ///   - title: The main title text to display.
+    ///   - additionalInformation: Optional text to display after the title.
+    ///   - showToolTipImage: Indicates whether to show the tooltip icon.
```

Comment:
here also `showToolTipImage` should be `showTooltipImage`, and the other occurances